README.md

```diff
@@ -144,20 +144,25 @@ number.
 
 ### Caching dependencies downloaded by Deno automatically
 
-Dependencies installed by Deno can be cached automatically, which is similar to
-the [`cache` option in `setup-node`](https://github.com/actions/setup-node).
+Dependencies installed by Deno can be cached automatically.
 
-To enable the cache, use `cache: true`. It's recommended to also add the
-`cache-hash` property, to scope caches based on lockfile changes.
+To enable the cache, use `cache: true`.
 
 ```yaml
 - uses: denoland/setup-deno@v2
   with:
     cache: true
-    cache-hash: ${{ hashFiles('**/deno.lock') }}
 ```
```

Comment:
Maybe add an example of using `cache-hash` instead of `cache` after the first example here?

```md
It is possibly customize the default hash used as part of the cache key (`**/deno.lock`). This can be done via
the `cache-hash` input. Setting `cache-hash` implies `cache: true`.

``yaml
- uses: denoland/setup-deno@v2
  with:
    cache-hash: ${{ hashFiles('/deno.json') }}
``
```